pkg/build/runner.go

```diff
@@ -283,7 +283,7 @@ func (r *BuildRunner) RunBuildProcess(config BuildProcessConfig) {
 		config.BuildLogger.Write([]byte(errMsg + "\n"))
 	}
 
-	config.BuildLogger.Write([]byte("\nBuild completed successfully\n"))
+	config.BuildLogger.Write([]byte("\n \n\033[1mBuild completed successfully\033[0m\n"))
```

Comment:
I've seen Toma extracting these ANSI escape codes to their own variables and using them with `fmt.Sprintf(...)`.
Maybe we should consider extracting them globally to the common.go in views or something similar? I think that would be neat, but I'm open to either approach.

Reply:
Agreed, can't we use lipgloss for this or something?